lib/accessories/thermostat.js

```diff
@@ -88,21 +94,10 @@ FritzThermostatAccessory.prototype.getTargetHeatingCoolingState = function(callb
     callback(null, service.fritzTargetHeatingCoolingState);
 
     this.platform.fritz('getTempTarget', this.ain).then(function(temp) {
-        var state;
-
-        switch (temp) {
-            case 'off':
-                state = Characteristic.TargetHeatingCoolingState.OFF;
-                break;
-            case 'on':
-                state = Characteristic.TargetHeatingCoolingState.HEAT;
-                break;
-            default:
-                state = Characteristic.TargetHeatingCoolingState.AUTO;
-        }
+        const state = temp === 'off'? Characteristic.TargetHeatingCoolingState.OFF: Characteristic.TargetHeatingCoolingState.HEAT;
 
         service.fritzTargetHeatingCoolingState = state;
-        service.getCharacteristic(Characteristic.TargetHeatingCoolingState).setValue(state, undefined, FritzPlatform.Context);
+        service.getCharacteristic(Characteristic.TargetHeatingCoolingState).updateValue(state);
```

Comment:
I'm changing this as part of #80- you could check if that would work for you.

Reply:
I didn't dig into #80 
I wait until that gets merged and will then adjust for that in this PR and solve any possible merge conflicts.

Reply:
Any time frame when #80 is gonna be merged